internal/receiver/smartagentreceiver/README.md

```diff
@@ -20,17 +20,20 @@ should be used.
 1. Monitors with [dimension property and tag update
 functionality](https://dev.splunk.com/observability/docs/datamodel#Creating-or-updating-custom-properties-and-tags)
 allow an associated `dimensionClients` field that references the name of the SignalFx exporter you are using in your
-pipeline.  If you do not specify any exporters via this field, the receiver will attempt to use the associated
-pipeline.  If the next element of the pipeline isn't compatible with dimension update behavior, if a lone SignalFx
+pipeline.  These monitors include `ecs-metadata`, `heroku-metadata`, `kubernetes-cluster`, `openshift-cluster`, `postgresql`,
+and `sql`.
+If you do not specify any exporters via this field, the receiver will attempt to use the associated
+pipeline.  If the next element of the pipeline isn't compatible with dimension update behavior and if a lone SignalFx
 exporter was configured for your deployment, it will be selected.  If no dimension update behavior is desired,
 you can specify the empty array `[]` to disable.
```

Comment:
Suggested change: "If you do not specify any exporters via this field, the receiver will attempt to use the associated pipeline. If the next element of the pipeline isn't compatible with the dimension update behavior, and if you configured a single SignalFx exporter for your deployment, it would be selected. "